src/pages/submit.js

```diff
@@ -1,10 +1,210 @@
-import * as React from 'react';
-import Layout from '../components/Layout';
-
-const Submit = () => (
-  <Layout>
-    <h1>This is the submit work page!</h1>
-  </Layout>
-);
+import * as React from "react";
+import { graphql, useStaticQuery } from "gatsby";
+import Layout from "../components/Layout";
+import { H2, YELLOW, MarginContainer } from "../styles/styles";
+import { PrismicRichText } from "@prismicio/react";
+import {
+  HeaderImage,
+  GetInvolvedSection,
+  QuestionsSection,
+  QuestionsText,
+  GetInvolvedText,
+  ContentContainer,
+  InformationContainer,
+  FormContainer,
+  FormRow,
+  Input,
+  TextArea,
+  SubmitDescription,
+  HiddenInput,
+  UploadButton,
+  UploadButtonText,
+  UploadIcon,
+  PaddedInput,
+  SubmitContainer,
+} from "../styles/submit.styles";
+import Button from "../components/Button";
+
+const Submit = () => {
+  const submitQuery = useStaticQuery(graphql`
+    query submitQuery {
+      prismicSubmitWorkPage {
+        data {
+          call_to_action {
+            richText
+          }
+          credits_label {
+            text
+          }
+          email_label {
+            text
+          }
+          equipment_label {
+            text
+          }
+          get_involved {
+            richText
+          }
+          header_image {
+            url
+            alt
+          }
+          name_label {
+            text
+          }
+          project_description_label {
+            text
+          }
+          project_medium_label {
+            text
+          }
+          project_title_label {
+            text
+          }
+          questions {
+            richText
+          }
+          submit_button_label {
+            text
+          }
+          submit_intro {
+            richText
+          }
+          submit_title {
+            text
+          }
+          upload_button_label {
+            text
+          }
+        }
+      }
+    }
+  `);
+
+  const {
+    prismicSubmitWorkPage: {
+      data: {
+        call_to_action: callToAction,
+        credits_label: creditsLabel,
+        email_label: emailLabel,
+        equipment_label: equipmentLabel,
+        get_involved: getInvolved,
+        header_image: headerImage,
+        name_label: nameLabel,
+        project_description_label: projectDescriptionLabel,
+        project_medium_label: projectMediumLabel,
+        project_title_label: projectTitleLabel,
+        questions: questions,
```

Comment:
small unnecessary reassignment warning caused by this line
<img width="678" alt="Screen Shot 2022-05-04 at 3 47 48 PM" src="https://user-images.githubusercontent.com/55252799/166814405-31a0be81-63e6-4ee5-b6b5-1c48c0d30cdc.png">